perses/rjmc/topology_proposal.py

```diff
@@ -797,7 +797,7 @@ def __init__(self,
                  logp_proposal=None,
                  new_to_old_atom_map=None, old_alchemical_atoms=None,
                  old_chemical_state_key=None, new_chemical_state_key=None,
-                 old_residue_name=None, new_residue_name=None,
+                 old_residue_name='MOL', new_residue_name='MOL',
```

Comment:
Do we really want to change the default here, rather than just in `relative_setup.py`? 

Could this have surprising behavior for other code, or break any other tests? 

Would it be better to require these be specified or raise an exception if either one is `None`?

Reply:
It looks like it doesn't break any tests, so it's probably fine for now!

Reply:
I'm happy to make these required if that's better?

Reply:
How about we open a new issue to remind us to consider doing that later and just move on now?